Stripe/StripeiOS/Source/STPPinManagementService.swift

```diff
@@ -13,6 +13,7 @@ import Foundation
 import UIKit
 
 /// STPAPIClient extensions to manage PIN on Stripe Issuing cards
+@available(*, deprecated, message: "Please use Issuing Elements instead: https://stripe.com/docs/issuing/elements")
```

Comment:
1. The link leads to a page which seems to focus on a web integration?
2. I would expect that the deprecation message to tell users what other class/integration point to use?

Reply:
Yeah that's intended, we don't have an native Issuing Elements yet so this is the only option at the moment. 

Please https://jira.corp.stripe.com/browse/RUN_ISSUING_TED-2233 for context